common/compat.cppo.ml

```diff
@@ -0,0 +1,24 @@
+(* Copyright (C) 2019  Petter A. Urkedal <paurkedal@gmail.com>
+ *
+ * This library is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version, with the OCaml static compilation exception.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *)
+
+#if OCAML_VERSION < (4, 6, 0)
+module List = struct
+  include List
+  let init n f =
+    let rec loop acc i = if i < 0 then acc else loop (f i :: acc) (i - 1) in
+    loop [] (n - 1)
+end
+#endif
```

Comment:
Well, you don't need cppo here, you can just shadow or use `stdlib-shims`

Reply:
Thanks for you opinion, I'll shadow it. I though about `stdlib-shims`, as well, but it only provides a `Stdlib` module containing module alias for the top-level modules as they are in the current standard library.